src/components/Questions.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import icon from '../img/icon-arrow-down.svg';
+
+function Questions() {
+	return (
+		<div className="questions">
+			<ul className="questions__list">
+				<li className="questions__item">
+					<QuestionItem $id="1"
```

Comment:
instead of duplication you can do 
```
const questions = [
	{
		question: 'que ota?',
		answer: 'yes',
	},
	{
		question: 'que ota ota?',
		answer: 'no',
	},
];

function Questions() {
	const renderQuestion = (item, idx) => (
		<li>
			<QuestionItem
				question={item.question}
				answer={item.answer}
				key={`${item.question}_${item.answer}`}
				id={idx + 1}
			/>
		</li>
	);

	return (
		<ul>
			{questions.map(renderQuestion)}
		</ul>
	);
}
```